plugin/info.xml

```diff
@@ -4,7 +4,7 @@
 	<type>plugin</type>
 
 	<name>AirConsole</name>
-	<version>1.3</version>
+	<version>1.5.0.3</version>

```

Comment:
Well, I wanted it that way because:
1.5.0 is your side, airconsole-x.x.x.js. So in case you update to 1.5.1 for any kind of hotfix, we can update that. But what if we update the plugin itself? That's the forth digit. If not wanted, I can remove it in the next PR, coming with isUserLoggedIn and other things